Cargo.toml

```diff
@@ -28,11 +28,13 @@ futures = "0.1.11"
 futures-cpupool = "0.1"
 hyper = { git = "https://github.com/hyperium/hyper", optional = true }
 hyper-tls = { git = "https://github.com/hyperium/hyper-tls", optional = true }
+jsonwebtoken = { version = "2.0", optional = true }
 libc = "0.2.10"
 local-encoding = "0.2.0"
 log = "0.3.6"
 lru-disk-cache = { path = "lru-disk-cache" }
 number_prefix = "0.2.5"
+openssl = { version = "0.9", optional = true }
```

Comment:
That's unfortunate that pem-parser doesn't handle invalid files properly. If you feel sufficiently motivated you could submit a PR to that project to fix it. :) I understand if you want to skip that, I've been on the other end of this process and it can feel like you wind up fixing every part of the stack just to fix the one thing you want.